sebex/cmd/release.py

```diff
@@ -30,25 +31,70 @@ def status():
     else:
         success('There is no release pending at this moment, feel free to start one.')
 
+def gather_input() -> dict[Version, ProjectHandle]:
+    bumps = {}
+    log("hit enter when done", color='yellow')
+    while True:
+        value = click.prompt("Project", default="", show_default=False)
+        if value == "":
+            break
+        project = valid_project(value)
+        if project is None:
+            continue
```

Comment:
frankly I would suggest sticking with click options/prompts instead of doing app input manually. The interface that I imagined was to allow _adding_ project to existing release plan with subsequent sebex command runs instead of doing everything in single run.

Reply:
Do you have a suggestion for how to achieve prompting for projects until a blank line is passed with click options? `@click.option(multiple=True, prompt=True)` is [weird](https://github.com/pallets/click/issues/218)